src/DIRAC/WorkloadManagementSystem/Agent/PilotStatusAgent.py

```diff
@@ -38,8 +39,8 @@ class PilotStatusAgent(AgentModule):
                    for the agent restart
   """
 
-  queryStateList = ['Ready', 'Submitted', 'Running', 'Waiting', 'Scheduled']
-  finalStateList = ['Done', 'Aborted', 'Cleared', 'Deleted', 'Failed']
+  queryStateList = [PilotStatus.SUBMITTED, PilotStatus.RUNNING, PilotStatus.WAITING, 'Scheduled']
```

Comment:
So, `Ready` does not exist anymore, correct?

Reply:
From what I understand it does not exist anymore.
I did not find any pilot with this status, it does not exist in the code except in the `PilotStatusAgent/SiteDirector/PilotManagerHandler` lists.